apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -760,6 +831,25 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
             None if num_initial_correct == len(submission) else num_initial_correct
         )
 
+        if feedback_type in [
```

Comment:
An identical check to this appears in a couple places; should just set to a variable.

Reply:
fixed